templates/paragraphs/paragraph--oe-facts-figures.html.twig

```diff
@@ -0,0 +1,21 @@
+{#
+/**
+ * @file
+ * Theme override to display the 'Facts and Figures' #type paragraph.
+ *
+ * @see ./modules/contrib/paragraphs/templates/paragraph.html.twig
+ */
+#}
+{{ pattern('fact_figures', {
+  'style': style,
+  'title': title,
+  'items': items,
+  'display_icons': display_icons,
+  'responsive_columns': responsive_columns,
+  'grid_columns': grid_columns,
+  'gutter': gutter,
+  'equal_height': equal_height,
```

Comment:
don't pass these items where you added defaults, this needs to be handled in the pattern template